src/InstrumentationEngine/ProfilerManager.cpp

```diff
@@ -52,7 +52,7 @@ CProfilerManager::CProfilerManager() :
         tstringstream mboxStream;
         DWORD pid = GetCurrentProcessId();
         mboxStream << _T("MicrosoftInstrumentationEngine ProfilerAttach. PID: ") << pid;
-        MessageBox(NULL, mboxStream.str().c_str(), L"", MB_OK);
```

Comment:
>MessageBox [](start = 8, length = 10)

Interesting, I'd assume this would have redirected to MessageBoxA or MessageBoxW internally

Reply:
it does, there's a define statement in a Windows header file that picks the right one, but prefast was complaining - it wanted the actual exported name.